testing/test_deploy.py

```diff
@@ -289,6 +298,122 @@ def ks_deploy(app_dir, component, params, env=None, account=None):
     apply_command.append("--as=" + account)
   util.run(apply_command, cwd=app_dir)
 
+def modify_minikube_config(config_path, certs_dir):
```

Comment:
what's the purpose of doing this?

Reply:
We need to make the config work inside the pod and not the VM where minikube is running.
Added a comment.

Reply:
Thanks!